libcloud/storage/providers.py

```diff
@@ -37,6 +37,8 @@
     ('libcloud.storage.drivers.s3', 'S3USGovWestStorageDriver'),
     Provider.S3_CN_NORTH:
     ('libcloud.storage.drivers.s3', 'S3CNNorthStorageDriver'),
+    Provider.S3_CN_NORTHQWEST:
```

Comment:
Was this Q intended? Or did you perhaps mean `S3_CN_NORTHWEST` instead of `S3_CN_NORTHQWEST` here?

Reply:
sorry, it‘s spelling mistakes，i will add the test and check my code repeatedly；
And, in my company, the s3 service is in cn-northwest region, the technicist of aws told us  servers  in this region will raise continuously. without that change,  using libcloud to access s3 object in cn-northwest won't work. so this is a important deal for us! Sorry for my mistake again!